lib/ansible/modules/cloud/azure/azure_rm_routetable.py

```diff
@@ -73,13 +72,17 @@
 '''
 RETURN = '''
 changed:
```

Comment:
While we're updating this documentation, I would remove the entire entry for `changed` from the `RETURN` section - `changed` is not returned by the module, but by Ansible core, so it doesn't really belong in the module documentation.

Reply:
Thank you for your review. I will pay attention to this problem in future maintenance. Thanks a lot!